GANDLF/training_loop.py

```diff
@@ -85,12 +85,13 @@ def step(model, image, label, params):
             output = model(image)
     else:
         output = model(image)
-    
-    
+
     if "medcam_enabled" in params and params["medcam_enabled"]:
         output, attention_map = output
 
-    # print("Output shape : ", output.shape, flush=True)
+    print("Output shape : ", output.shape, flush=True)
+    print("label shape : ", label.shape, flush=True)
```

Comment:
If this is something for debugging, please remove, or if this needs to be presented to the user, please under verbose flag like this:

```suggestion
    if params["verbose"]:
        print("Output shape : ", output.shape, flush=True)
        print("label shape : ", label.shape, flush=True)
```

Reply:
Thanks @sarthakpati! it was for debugging and I will remove completely